knowledge/musical_instruments/les_paul_studio/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: musical_instruments
+seed_examples:
+- answer: |Various sunbursts,Translucent Red,Translucent Blue, Translucent Amber, Wine Red, Ebony, Alpine White
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!